notes/orca-architecture.md

```
@@ -0,0 +1,215 @@
+# Orca 项目包结构说明
+
+## 顶层目录
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)